app/cdap/components/GridTable/index.tsx

```diff
@@ -57,7 +65,9 @@ export default function GridTable() {
     },
   ]);
   const [showBreadCrumb, setShowBreadCrumb] = useState(true);
-
+  const [directiveFunction, setDirectiveFunction] = useState('');
+  const [directiveFunctionSupportedDataType, setDirectiveFunctionSupportedDataType] = useState([]);
```

Comment:
add types for states, directiveFunction and directiveFunctionSupportedDataType

Reply:
done